commands/station.js

```diff
@@ -32,8 +32,13 @@ export const station = async ({ json, experimental }) => {
   ) {
     panic('FIL_WALLET_ADDRESS must start with f410 or 0x')
   }
-  const fetchRes = await fetch(
-    `https://station-wallet-screening.fly.dev/${FIL_WALLET_ADDRESS}`
+  const fetchRes = await pRetry(
+    () => fetch(`https://station-wallet-screening.fly.dev/${FIL_WALLET_ADDRESS}`),
+    {
+      retries: 1000,
+      onFailedAttempt: () =>
+        console.error('Failed to check FIL_WALLET_ADDRESS address. Retrying...')
```

Comment:
Nitpick, feel free to ignore:

IMO, "Failed to validate... " would be a more accurate message than "Failed to check..."

Reply:
https://github.com/filecoin-station/core/pull/268